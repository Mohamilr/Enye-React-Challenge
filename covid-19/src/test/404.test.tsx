import React from 'react';
import { render } from '@testing-library/react';
import NotPage from '../components/404';

test('render landing page', () => {
    const { asFragment, getByText } =  render(<NotPage />)
    const h1 = getByText(/Page not found/i);
    const link = getByText(/Go back home/i);

  expect(asFragment()).toMatchSnapshot();
  expect(h1).toBeInTheDocument();
  expect(link).toBeInTheDocument();
})