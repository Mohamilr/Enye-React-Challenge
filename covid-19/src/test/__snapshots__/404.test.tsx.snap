// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render landing page 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-parentDiv-8"
  >
    <h1>
      Page not found
    </h1>
    <a
      href="/"
    >
      Go back home
    </a>
  </div>
</DocumentFragment>
`;
